src/test/scala/org/ua/ilm/datamining/crawlers/youtube/CrawlerChannelsTest.scala

```diff
@@ -8,10 +8,10 @@ class CrawlerChannelsTest extends org.scalatest.FunSuite with BeforeAndAfterAllC
 
   test("CrawlerChannels.channelListRequest") {
```

Comment:
there are a couple of possible tests for this method. add ehat are you actually checking to test name please